apache-pulsar/src/main/java/io/debezium/examples/apache/pulsar/PulsarProducer.java

```diff
@@ -0,0 +1,171 @@
+/*
+ * Copyright Debezium Authors.
+ *
+ * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.debezium.examples.apache.pulsar;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.text.MessageFormat;
+import java.util.HashMap;
+import java.util.Properties;
+import java.util.concurrent.ExecutorService;
+import java.util.concurrent.Executors;
+import java.util.concurrent.TimeUnit;
+
+import org.apache.kafka.connect.json.JsonConverter;
+import org.apache.kafka.connect.source.SourceRecord;
+import org.apache.pulsar.client.api.MessageId;
+import org.apache.pulsar.client.api.Producer;
+import org.apache.pulsar.client.api.PulsarClient;
+import org.apache.pulsar.client.api.PulsarClientException;
+import org.apache.pulsar.client.api.Schema;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import io.debezium.config.Configuration;
+import io.debezium.embedded.EmbeddedEngine;
+import io.debezium.examples.apache.pulsar.config.PropertyLoader;
+import io.debezium.util.Clock;
+
+public class PulsarProducer implements Runnable {
+
+    private static final Logger logger = LoggerFactory.getLogger(PulsarProducer.class);
+
+    private final Configuration config;
+    private final JsonConverter valueConverter;
+    private final JsonConverter keyConverter;
+    private PulsarClient client;
+    private final HashMap<String, Producer<String>> producerHashMap = new HashMap<>();
```

Comment:
Neitther type nor name should contain the implementation of the map, so the hash should be dropped

Reply:
Ok :)